src/opengeodeweb_viewer/rpc/utils_protocols.py

```diff
@@ -25,3 +29,40 @@ def kill(self) -> None:
         )
         print("Manual viewer kill, shutting down...", flush=True)
         os._exit(0)
+
+    @exportRpc(utils_prefix + utils_schemas_dict["import_project"]["rpc"])
+    def importProject(self, rpc_params: ImportProject) -> None:
```

Comment:
```suggestion
    def importProject(self, rpc_params: RPCParams) -> None:
```
c'est pas ca ? et tu peux supprimer le cast plus bas